test/e2e/api-specs/MultichainAuthorizationConfirmationErrors.ts

```diff
@@ -74,7 +74,7 @@ export class MultichainAuthorizationConfirmationErrors implements Rule {
     if (isMethodAllowed) {
       if (method.errors) {
         method.errors.forEach((err) => {
-          const unsupportedErrorCodes = [5000, 5300, 5301];
+          const unsupportedErrorCodes = [5000, 5100, 5101, 5102, 5300, 5301];
```

Comment:
we should update the spec to remove these errors. Or if we are adding them as warnings we should be able to write a new `Rule` to test for that based on the errors in the spec.